CMakeLists.txt

```diff
@@ -11,8 +11,16 @@ set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
 include(FetchContent)
 
+option(SWIFT_FIREBASE_BUILD_EXAMPLES "Build example UI" TRUE)
+
+# ${TOOLCHAIN_ROOT}/usr/include allows us to access swift C++ interop headers.
+cmake_path(GET CMAKE_Swift_COMPILER PARENT_PATH _SWIFT_INCLUDE_DIR)
+cmake_path(GET _SWIFT_INCLUDE_DIR PARENT_PATH _SWIFT_INCLUDE_DIR)
+include_directories(SYSTEM ${_SWIFT_INCLUDE_DIR}/include)
+
 add_library(firebase INTERFACE)
 target_compile_options(firebase INTERFACE
+  "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xcc -DSR69711>"
```

Comment:
It might be just me, but does `SR69711` warrant some comment? It's kinda cryptic - at least for me, and even google search doesn't really give much information about it.

Reply:
I don't think it really does - this is basically the same as WPP-????. It is a bug report identifier (https://github.com/apple/swift/issues/69711).